plugins/template/MS_Template/template.py

```diff
@@ -0,0 +1,120 @@
+import datetime, requests
```

Comment:
All comments from previous template apply here

Curious if we could make this ultra generic, where you just specify the resource? Picking from vault.azure.net, graph.microsoft.net, etc so there doesn't need to be templates for each